lib/components/Input/input-phone-number/country-flags.tsx

```diff
@@ -1,503 +1,505 @@
-    FlagSudanIcon,
-    FlagSurinameIcon,
-    FlagSvalbardAndJanMayenIcon,
-    FlagSwazilandIcon,
-    FlagSwedenIcon,
-    FlagSwitzerlandIcon,
-    FlagSyriaIcon,
-    FlagTaiwanIcon,
-    FlagTajikistanIcon,
-    FlagTanzaniaIcon,
-    FlagThailandIcon,
-    FlagTimorLesteIcon,
-    FlagTogoIcon,
-    FlagTokelauIcon,
-    FlagTongaIcon,
-    FlagTrinidadAndTobagoIcon,
-    FlagTunisiaIcon,
-    FlagTurkeyIcon,
-    FlagTurkmenistanIcon,
-    FlagTurksAndCaicosIslandsIcon,
-    FlagTuvaluIcon,
-    FlagUSVirginIslandsIcon,
-    FlagUgandaIcon,
-    FlagUkraineIcon,
-    FlagUnitedArabEmiratesIcon,
-    FlagUnitedKingdomIcon,
-    FlagUsaIcon,
-    FlagUzbekistanIcon,
-    FlagVanuatuIcon,
-    FlagVaticanCityIcon,
-    FlagVenezuelaIcon,
-    FlagVietnamIcon,
-    FlagWallisAndFutunaIcon,
-    FlagWesternSaharaIcon,
-    FlagYemenIcon,
-    FlagZambiaIcon,
-    FlagZimbabweIcon,
+    // FlagPolandIcon,
+    // FlagPortugalIcon,
+    // FlagPuertoRicoIcon,
+    // FlagQatarIcon,
+    // FlagRUnionIcon,
+    // FlagRomaniaIcon,
+    // FlagRussiaIcon,
+    // FlagRwandaIcon,
+    // FlagSaintBarthelemyIcon,
+    // FlagSaintHelenaIcon,
+    // FlagSaintKittsAndNevisIcon,
+    // FlagSaintLuciaIcon,
+    // FlagSaintMartinIcon,
+    // FlagSaintPierreAndMiquelonIcon,
+    // FlagSaintVincentAndGrenadinesIcon,
+    // FlagSamoaIcon,
+    // FlagSanMarinoIcon,
+    // FlagSaoTomeAndPrincipeIcon,
+    // FlagSaudiArabiaIcon,
+    // FlagSeirraLeoneIcon,
+    // FlagSenegalIcon,
+    // FlagSerbiaIcon,
+    // FlagSeychellesIcon,
+    // FlagSingaporeIcon,
+    // FlagSintMaartenIcon,
+    // FlagSlovakiaIcon,
+    // FlagSloveniaIcon,
+    // FlagSolomonIslandsIcon,
+    // FlagSomaliaIcon,
+    // FlagSouthAfricaIcon,
+    // FlagSouthGeorgiaAndTheSouthSandwichIslandsIcon,
+    // FlagSouthKoreaIcon,
+    // FlagSouthSudanIcon,
+    // FlagSpainIcon,
+    // FlagSriLankaIcon,
+    // FlagSudanIcon,
+    // FlagSurinameIcon,
+    // FlagSvalbardAndJanMayenIcon,
+    // FlagSwazilandIcon,
+    // FlagSwedenIcon,
+    // FlagSwitzerlandIcon,
+    // FlagSyriaIcon,
+    // FlagTaiwanIcon,
+    // FlagTajikistanIcon,
+    // FlagTanzaniaIcon,
+    // FlagThailandIcon,
+    // FlagTimorLesteIcon,
+    // FlagTogoIcon,
+    // FlagTokelauIcon,
+    // FlagTongaIcon,
+    // FlagTrinidadAndTobagoIcon,
+    // FlagTunisiaIcon,
+    // FlagTurkeyIcon,
+    // FlagTurkmenistanIcon,
+    // FlagTurksAndCaicosIslandsIcon,
+    // FlagTuvaluIcon,
+    // FlagUSVirginIslandsIcon,
+    // FlagUgandaIcon,
+    // FlagUkraineIcon,
+    // FlagUruguayIcon,
+    // FlagUnitedArabEmiratesIcon,
+    // FlagUnitedKingdomIcon,
+    // FlagUsaIcon,
+    // FlagUzbekistanIcon,
+    // FlagVanuatuIcon,
+    // FlagVaticanCityIcon,
+    // FlagVenezuelaIcon,
+    // FlagVietnamIcon,
+    // FlagWallisAndFutunaIcon,
+    // FlagWesternSaharaIcon,
+    // FlagYemenIcon,
+    // FlagZambiaIcon,
+    // FlagZimbabweIcon,
 } from "@deriv/quill-icons/Flags";
```

Comment:
for readablity, can we move import statements together and comment out all other lines
```
import React, { ReactNode } from "react";
import { FlagPlaceholderIcon } from "@deriv/quill-icons/Flags";

export const FlagsMap: Record<string, ReactNode> = {
    XX: <FlagPlaceholderIcon />,
};
```